app/models/user.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 class User < ActiveRecord::Base
   has_many :versions, inverse_of: :user
```

Comment:
Every version action is initiated by a User, which then can be tracked. So if some question about who did what arises, we can determine it.